bridge/runbook/validator_runbook.md

```diff
@@ -56,3 +56,36 @@ Suggested hardware requirements:
 * Memory: 16GB
 * Storage: 200GB
 * Network: 100Mbps
+
+### Optional: WAF Protection for Sui Bridge Server
+
+It is a good idea to add some sort of rate limit protection to the bridge server which will be exposed to the public internet. 
+This will give node operators fine-grained control over the rate of requests the receive, and observability into those requests.
```

Comment:
This should be required based on our discussions on the threat model, i.e. we assume there is rate limiting in front of the node.